packages/web-components/src/player-component/player.class.ts

```diff
@@ -438,6 +438,46 @@ export class PlayerWrapper {
         }
     }
 
+    private disableNextSegmentButton(disable: boolean) {
+        for (const element of this.controls?.elements_) {
+            if (element?.isNextSegmentButton) {
+                element.disableNextSegmentButton(disable);
+            }
+        }
+    }
+
+    private disablePrevSegmentButton(disable: boolean) {
+        for (const element of this.controls?.elements_) {
+            if (element?.isPrevSegmentButton) {
+                element.disablePrevSegmentButton(disable);
+            }
+        }
+    }
+
+    private updateNextSegmentButton() {
+        const lastSegmentStartSeconds = extractRealTimeFromISO(
+            this._availableSegments?.timeRanges[this._availableSegments?.timeRanges.length - 1]?.start,
+            this._currentDate
+        );
+        if (lastSegmentStartSeconds === this.currentSegment.startSeconds) {
```

Comment:
this.disableNextSegmentButton(lastSegmentStartSeconds === this.currentSegment.startSeconds)

Reply:
Change is made.